slackcli/utils.py

```diff
@@ -74,7 +74,8 @@ def search_messages(source_name, count=20):
 
 def format_message(source_name, message):
     time = datetime.fromtimestamp(float(message['ts']))
+    name = names.username(message['user']) if ('user' in message and len(message['user']) > 0) else message['username']
```

Comment:
May I suggest to simplify as the following:

    username = message.get('username')
    if not username:
        username = names.username(message['user'])

Also, could you please add a comment in the code that explains that some bot messages contain a "username" key?